osm2pgsql.lua

```diff
@@ -499,128 +499,95 @@ function osm2pgsql.process_node(object)
         object.tags.wikidata == 'Q994730') then
         output_hstore['disputed_by'] = 'CN;RU;IN;GR'
     end
--- Recast Northern Cyprus as country label and turn off for several POVs including China and Russia
-    if object.tags.place and object.tags.wikidata == 'Q23681' then
+
+-- Recast various dependencies as countries
+-- American Samoa
+    if object.tags.place == 'state' and object.tags['ISO3166-1'] == 'AS' then
         output_hstore['place'] = 'country'
-        output_hstore['disputed_by'] = 'CN;RU;IN;GR;CY'
     end
--- Turn off Abkhazia label for most countries
```

Comment:
are these turned off in the style?  I'm not sure we should rely on them being absent in the style because things happen... maybe we should just keep these in as a belt-and-suspenders approach if we're confident we don't want to show them?  Just a thought

Reply:
Yeah, thats a good point. We specifically added some of these to our patch (and have now removed them) but some are or will be in Daylight. I'll update this to nil out the place for these instead.

Reply:
great

Reply:
We want to turn these off in the data... but like Northern Cyrpus that should happen automatically since Abkhazia isn't in the NE "countries" list.  So I'd revert this.